crawl-ref/source/mon-data.h

```diff
@@ -1643,6 +1643,19 @@ DUMMY(MONS_GIANT_LIZARD, 'l', LIGHTGREY, "giant lizard", TILEP_MONS_LEOPARD_GECK
     {TILEP_MONS_GHOUL}, TILE_CORPSE_GHOUL
 },
 
+{
+    MONS_VAUD, 'n', YELLOW, "Vaud",
+    M_UNIQUE | M_SPEAKS | M_SEE_INVIS | M_MALE | M_NO_ZOMBIE,
+    MR_RES_COLD,
+    22, MONS_GHOUL, MONS_VAUD, MH_UNDEAD, 50,
+    { {AT_HIT, AF_PLAIN, 16}, AT_NO_ATK, AT_NO_ATK, AT_NO_ATK },
+    5, 430,
+    2, 8, MST_NO_SPELLS, true, S_SHOUT,
```

Comment:
Given that the halo radius is only 3, it seems like an average player could just trivially and patiently pelt Vaud to death with stones. Is that what you were looking for?